Add option for Endpoint Security in Intune (#6563)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md

4. Select **OK** on the three configuration panes. Then select **Create** if you're creating a new endpoint protection file or **Save** if you're editing an existing one.

> **Endpoint Security Policy**

Reviewer: ```suggestion
**Endpoint security policy**
```

> 1. Select **Endpoint Security** \> **Attack surface reducation**. Choose an existing Attack surface reducation rule or create a new one. To create a new one, select **Create Policy** and enter information for this profile. For **Profile type**, select **Attach surface reducation rules**. If you've chosen an existing profile, select **Properties** and then select **Settings**.

Reviewer: ```suggestion
1. Select **Endpoint Security** \> **Attack surface reduction**. Choose an existing ASR rule or create a new one. To create a new one, select **Create Policy** and enter information for this profile. For **Profile type**, select **Attack surface reduction rules**. If you've chosen an existing profile, select **Properties** and then select **Settings**.
```

> 2. In the **Configuration settings** pane, select select **Attack Surface Reduction**. Select the desired setting for each ASR rule.

Reviewer: ```suggestion
2. In the **Configuration settings** pane, select **Attack Surface Reduction** and then select the desired setting for each ASR rule.
```

3. Under **List of additional folders that need to be protected**, **List of apps that have access to protected folders**, and **Exclude files and paths from attack surface reduction rules**, enter individual files and folders. You can also select **Import** to import a CSV file that contains files and folders to exclude from ASR rules. Each line in the CSV file should be formatted as follows:

   `C:\folder`, `%ProgramFiles%\folder\file`, `C:\path`

> 4. Select **Next** on the three configuration panes. Then select **Create** if you're creating a new policy or **Save** if you're editing an existing one.

Reviewer: ```suggestion
4. Select **Next** on the three configuration panes, then select **Create** if you're creating a new policy or **Save** if you're editing an existing policy.
```

> 

Reviewer: ```suggestion
```
